gc/verbose/VerboseHandlerOutput.hpp

```diff
@@ -242,6 +242,15 @@ class MM_VerboseHandlerOutput : public MM_Base
 	 */
 	void outputStringConstantInfo(MM_EnvironmentBase *env, uintptr_t indent, uintptr_t candidates, uintptr_t cleared);
 
+	/**
+	 * Output monitor references table summary.
+	 * @param env GC thread used for output.
+	 * @param indent base level of indentation for the summary.
+	 * @param candidates total count candidate monitor references considered
+	 * @param cleared total count of cleared monitor references
+	 */
+	void outputMonitorReferenceInfo(MM_EnvironmentBase *env, uintptr_t ident, uintptr_t candidates, uintptr_t cleared);
```

Comment:
Looks like function declaration still has a typo and outputStringConstantInfo declaration should be updated as well